src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -20229,9 +20229,15 @@ public static bool TakeOutParam<T>(T y, out T x)
 
                 var tree = compilation.SyntaxTrees.Single();
                 Assert.Empty(GetOutVarDeclarations(tree));
+                AssertNoGlobalStatements(tree);
```

Comment:
Oh, I see that you parsed the source twice, once in script mode and once without.

---
In reply to: [79447599](https://github.com/dotnet/roslyn/pull/13885#discussion_r79447599) [](ancestors = 79447599)